### DISCUSSION TITLE
What's the simplest way to disable a block or a segment?

### DISCUSSION THREAD

**Discussion by livioc** [JanDeDobbeleer/oh-my-posh#1215] [Troubleshoot] [answered]
Hi, is it possible to disable a single block or segment by configuration?
I'm new to omp and expected to have a property "enable" at block/segment level but I didn't found any so actually I'm just removing/adding the block or section from the configuration file while doing some test about customizing a theme but it's not very handy

thanks

**Comment by JanDeDobbeleer** [comment, marked as answer]
Removing it is indeed the only way. It's a bit weird to have segments in there that you don't want to use (at least that's the reasoning).

**Comment by livioc** [comment]
in my opinion there are some cases where it might be useful, for example keeping 'battery' disabled when you work at home/office and enable it only when you have to work in same place where you can't charge your laptop all the time. 
anyhow thanks for your reply and for omp, it's great :)

**Reply by JanDeDobbeleer** [reply]
That could bring us to another options as in only display that segment when it's charging. I like the idea of using the segment's context to define if it's enabled or not via templating (and not a gazillion switches). Something that's definitely worth the investigation when we're done with offering templates everywhere.
